docs/Initial-Setup/Setting-up-your-Manager-Instance.rst

```diff
@@ -20,44 +20,43 @@ right corner, ensure that the correct region is selected.
 
 To launch a manager instance, follow these steps:
 
-1. From the main page of the EC2 Management Console, click
+#. From the main page of the EC2 Management Console, click
```

Comment:
Until the new launch experience is required, I think we should mention that these instructions are for that experience. Maybe a warning?

Reply:
Good idea.

Reply:
https://github.com/firesim/firesim/pull/1036/files#diff-ca8616060cacf383a14a75e23f9ad117cccfe1c7bd2c5407eecc07349b9bce53R7

Reply:
https://github.com/firesim/firesim/pull/1036/files#diff-ca8616060cacf383a14a75e23f9ad117cccfe1c7bd2c5407eecc07349b9bce53R7